### PR TITLE
Update CyberArkIdentityEventCollector_description.md

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Update description file with the correct parameters for creating an instance of the integration.
In particular, removed Vendor Name and Product Name.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md [hunk 1]
```diff
@@ -5,16 +5,22 @@ Before you start, you need to get the tenant ID, app ID, username, and password.
...
-The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``.
+The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``
 
 **App ID**  
 The application ID to fetch the logs from.
 
-**User name and Password**    
+**User name and Password** 
+
 The SIEM user name and password.
```
Comment (nkanon) [inline Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md:15]: I would change this line to:
The user that was created in CyberArk for XSIAM integration.

File: Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md [hunk 2]
```diff
@@ -5,16 +5,22 @@ Before you start, you need to get the tenant ID, app ID, username, and password.
 ## Configuration Parameters
 
 **Server URL**    
-The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``.
+The endpoint to get the logs. For example, ``https://{{tenant}}.my.idaptive.app/``
```
Comment (nkanon) [inline Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md:8]: I would change this line to:
The CyberArk Identity URL to get the logs from. For example, ``https://{{tenant}}.my.idaptive.app/``

File: Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md [hunk 3]
```diff
@@ -5,16 +5,22 @@ Before you start, you need to get the tenant ID, app ID, username, and password.
 ## Configuration Parameters
```
Comment (nkanon) [inline Packs/CyberArkIdentity/Integrations/CyberArkIdentityEventCollector/CyberArkIdentityEventCollector_description.md:5]: The first line should be (Removed the "Event Collector"):
Use the CyberArk Identity integration to get Audit and Auth logs for an application using REST APIs.

**Comment by ShirleyDenkberg** [comment]
@nkanon Thanks for the input.  All comments implemented.

**Comment by ShirleyDenkberg** [comment]
@nkanon Please approve this PR so we can close it.

**Review by nkanon — APPROVED** [review]

**Pull request closed without merging**